fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -163,7 +163,8 @@ default void dropTable(DropTableStmt stmt) throws DdlException {
     default void alterTable(AlterTableStmt stmt) throws UserException {
     }
 
-    default void createTable(CreateTableStmt stmt) throws DdlException {
+    default boolean createTable(CreateTableStmt stmt) throws DdlException {
```

Comment:
Please comment when to return true or not.

Reply:
I write a comment in the implementation instead of here